frontend/src/assets/styles/themes.scss

```diff
@@ -0,0 +1,17 @@
+@use "sass:map";
+@use "@bcgov-nr/nr-theme/design-tokens/light-theme.scss" as light;
+@use "@bcgov-nr/nr-theme/design-tokens/dark-theme.scss" as dark;
+
+@use "@carbon/themes/scss/themes";
+@use "@carbon/themes/scss/theme" with (
+    $theme: light.$light-theme,
+    $fallback: themes.$white
+);
+
+@forward "@carbon/themes/scss/theme";
+
+:root {
+    @each $key, $value in light.$light-theme {
+        --#{$key}: #{$value};
+    }
+}
```

Comment:
good question, my goal is to get rid of the `default-theme.scss` and the usage of the primevue-override from the nr-theme package in the responsiveness ticket for the next sprint. I didn't remove the default theme because it will create even more work to refactor and it was too overwhelming for me